src/Grpc.AspNetCore.Server/Internal/HttpContextServerCallContext.Log.cs

```diff
@@ -41,6 +41,9 @@ private static class Log
             private static readonly Action<ILogger, string, Exception> _encodingNotInAcceptEncoding =
                 LoggerMessage.Define<string>(LogLevel.Debug, new EventId(5, "EncodingNotInAcceptEncoding"), "Request grpc-encoding header value '{GrpcEncoding}' is not in grpc-accept-encoding.");
 
+            private static readonly Action<ILogger, Exception> _deadlineCancellationError =
+                LoggerMessage.Define(LogLevel.Error, new EventId(6, "DeadlineCancellationError"), "Error canceling request because of the deadline.");
```

Comment:
nit: the error message is not very readable.

Reply:
I think the meaning is something like "Error occurred while trying to cancel the request due to deadline exceeded"